airflow/hooks/S3_hook.py

```diff
@@ -272,7 +272,7 @@ def select_key(self, key, bucket_name=None,
             InputSerialization=input_serialization,
             OutputSerialization=output_serialization)
 
-        return ''.join(event['Records']['Payload']
+        return ''.join(event['Records']['Payload'].decode('utf-8')
```

Comment:
This possibly needs to be a parameter (with a default value of `utf-8`) just in case not everyone's data is UTF-8

Reply:
There are two other hard-coded "utf-8" in S3_hook file, one of them in "read_key" function with same decode.
Changing it to parameter is much bigger scope that this PR

Reply:
Ah okay, fair point.

Reply:
@ashb is that fix will be available in next release?

Reply:
Should be more - I've marked it for inclusion in the release